.github/workflows/hypertrace-ingester/docker-compose.yml

```diff
@@ -47,6 +47,8 @@ services:
       - REPLICATION_FACTOR=1
       - ENTITY_SERVICE_HOST_CONFIG=hypertrace
       - ENTITY_SERVICE_PORT_CONFIG=9001
+      - ATTRIBUTE_SERVICE_HOST_CONFIG=hypertrace
+      - ATTRIBUTE_SERVICE_PORT_CONFIG=9001
```

Comment:
same port?

Reply:
the e2e test uses hypertrace-service the mega-image, so both entity and attribute services are colocated